privval/msgs_test.go

```diff
@@ -71,7 +71,7 @@ func TestPrivvalVectors(t *testing.T) {
 	proposal := exampleProposal()
 	proposalpb := proposal.ToProto()
 
-	// Create a Reuseable remote error
+	// Create a Reusable remote error
```

Comment:
[optional] IMO this shouldn't be capitalized
```suggestion
	// Create a reusable remote error
```

Reply:
yeah... I'll fix that as well.